pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -148,9 +148,9 @@ type InTreeToCSITranslator interface {
 // 2. Once all the assume operations are done in e), the scheduler processes the next Pod in the scheduler queue
 //    while the actual binding operation occurs in the background.
 type SchedulerVolumeBinder interface {
-	// GetPodVolumes returns a pod's PVCs separated into bound, unbound with delayed binding (including provisioning)
-	// and unbound with immediate binding (including prebound)
-	GetPodVolumes(pod *v1.Pod) (boundClaims, unboundClaimsDelayBinding, unboundClaimsImmediate []*v1.PersistentVolumeClaim, err error)
+	// GetPodVolumes returns a pod's PVCs separated into bound, lost,
```

Comment:
Can we put the logic in https://github.com/kubernetes/kubernetes/blob/59fde306060fa7762d87feb80cbaee948361c507/pkg/scheduler/framework/plugins/volumebinding/volume_binding.go#L126 instead?

Reply:
Thanks for the suggestion! Moved the additional check to `podHasPVCs`

Reply:
@msau42 I've made the change as suggested, can you have another look whenever you get a chance? Thanks!